site/content/3.12/develop/http-api/collections.md

```diff
@@ -950,6 +950,57 @@ logJsonResponse(response);
 db._drop(cn);
 ```
 
+### Get the available key generators
+
+```openapi
+paths:
+  /_api/key-generators:
+    get:
+      operationId: getKeyGenerators
+      description: |
+        Returns the available key generators for collections.
+
+        The call returns a JSON object with the available key generators
+        on success:
+
+        - `padded`
+        - `uuid`
+        - `autoincrement`
+        - `traditional`
+      responses:
+        '200':
```

Comment:
@Simran-B I applied your suggestion but the output is not right, see https://deploy-preview-420--docs-hugo.netlify.app/devel/develop/http-api/collections/#get-the-available-key-generators_res_200

Reply:
Seems fine in Swagger UI and swagger-cli validated it as correct, so it looks like the toolchain is not able to render a construct like this

<img width="420" alt="image" src="https://github.com/arangodb/docs-hugo/assets/7819991/67d951d7-e2ea-4817-8fa2-4142d81d0efa">

Reply:
With added example value:
<img width="440" alt="image" src="https://github.com/arangodb/docs-hugo/assets/7819991/4030a4b3-ef53-4061-bfe1-1cc54760270f">

Reply:
Looks great!